src/com/company/exercises/Demo.java

```diff
@@ -0,0 +1,36 @@
+package com.company.exercises;
+
+public class Demo {
+    @FunctionalInterface
```

Comment:
It is generally accepted to have an empty line between the start of a class and the code within it.

Reply:
Done.